mojave/items/clothing/under/wasteland/under.dm

```diff
@@ -196,3 +196,81 @@
 	desc = "A rugged plaid shirt."
 	icon_state = "plaid_b"
 	inhand_icon_state = "plaid_b"
+
+/obj/item/clothing/under/ms13/prewar/checkeredred
+	name = "red checkered shirt"
+	desc = "A shirt with a red checker pattern."
+	icon_state = "checkred"
+	inhand_icon_state = "plaid_r"
+
+/obj/item/clothing/under/ms13/wasteland/checkeredgreen
+	name = "green checkered shirt"
+	desc = "A shirt with a green checker pattern."
+	icon_state = "checkgreen"
+	inhand_icon_state = "plaid_g"
+
+/obj/item/clothing/under/ms13/wasteland/checkeredblue
+	name = "blue checkered shirt"
+	desc = "A shirt with a blue checkered pattern."
+	icon_state = "checkblue"
+	inhand_icon_state = "plaid_b"
+
+/obj/item/clothing/under/ms13/wasteland/stripered
+	name = "red striped shirt"
+	desc = "A red shirt with columns of white stripes."
+	icon_state = "stripered"
+	inhand_icon_state = "plaid_r"
+
+/obj/item/clothing/under/ms13/wasteland/stripeblue
+	name = "blue striped shirt"
+	desc = "A blue shirt with columns of white stripes."
+	icon_state = "stripeblue"
+	inhand_icon_state = "plaid_b"
+
+/obj/item/clothing/under/ms13/wasteland/brownvest
+	name = "brown vest"
+	desc = "A brown vest."
+	icon_state = "vestbrown"
+	inhand_icon_state = "lb_suit"
+
+/obj/item/clothing/under/ms13/wasteland/burgundyvest
+	name = "burgundy vest"
+	desc = "A burgundy vest."
+	icon_state = "vestbrown"
+	inhand_icon_state = "brownjsuit"
+
+/obj/item/clothing/under/ms13/wasteland/whiteshirt
+	name = "white shirt"
+	desc = "A plain white shirt."
+	icon_state = "shirt"
+	inhand_icon_state = "bar_suit"
+
+/obj/item/clothing/under/ms13/mechanicprewar
+	can_adjust = TRUE
+	alt_covers_chest = FALSE
+
+/obj/item/clothing/under/ms13/mechanicprewar/mechanicgrey/alt
+	icon_state = "mechanicg_d"
+
+/obj/item/clothing/under/ms13/mechanicprewar/mechanicgrey
+	name = "grey mechanic shirt"
+	desc = "A grey mechanic shirt. It smells of oil."
+	icon_state = "mechanicg"
+	inhand_icon_state = "bl_suit"
+	alternate_worn_layer = "mechanicg"
+	can_adjust = TRUE
+	alt_covers_chest = FALSE
+	alternate_appearances = "mechanicg_d"
+
+/obj/item/clothing/under/ms13/mechanicprewar/mechanicgreen/alt
+	icon_state = "mechanicalt_d"
+
+/obj/item/clothing/under/ms13/mechanicprewar/mechanicgreen
+	name = "green mechanic shirt"
+	desc = "A green mechanic shirt. It smells of oil, but is surprisingly clean."
+	icon_state = "mechanicalt"
+	inhand_icon_state = "g_suit"
+	alternate_worn_layer = "mechanicalt_d"
+	alternate_appearances = "mechanicalt_d"
+	can_adjust = TRUE
+	alt_covers_chest = FALSE
```

Comment:
It likely isn't worthwhile to add an `alt` object path since that means we have a whole separate object just for an adjusted icon state.
`alternate_appearances` no longer seems to be used in tgstation code for this sort of functionality and can be removed.
`alternate_worn_layer` is specifically for moving the layer of the clothing sprite which doesn't seem to be needed in this case.  You'd only use it if you wanted to either cover something (like the ghost bedsheet covering all clothing) or if you wanted to layer it under something.  Plus, the described layers are just icon states not actual layers, so they don't do anything at the moment.
The appended '_d' on the icon state should be handled by the code already when setting `can_adjust` to TRUE, so as long as that icon state exists in the .dmi, you should be fine.
Try testing it out, it should reduce the amount of unneeded code.

Actually, here is the code snippet for how alt appearances work inside the `/mob/living/carbon/human/update_inv_w_uniform()` proc
```
var/target_overlay = U.icon_state
		if(U.adjusted == ALT_STYLE)
			target_overlay = "[target_overlay]_d"
		else if(U.adjusted == DIGITIGRADE_STYLE)
			target_overlay = "[target_overlay]_l"
```